docs/route/route.md

```diff
@@ -205,10 +205,27 @@ let { org, "*": splat } = params;
 
 ### Layout Routes
 
-<docs-info>TODO: expand with example</docs-info>
-
 Omitting the path makes this route a "layout route". It participates in UI nesting, but it does not add any segments to the URL.
 
+```tsx
+  <MemoryRouter initialEntries={["/"]}>
+    <Routes>
+      <Route
+        element={
+          <div>
+            <h1>Layout</h1>
+            <Outlet />
+          </div>
+        }
+      >
+        <Route path="/home" element={<h1>Home</h1>} />
+      </Route>
+      <Route index element={<h1>Index</h1>} />
+    </Routes>
+  </MemoryRouter>
```

Comment:
```suggestion
<Route
  element={
    <div>
      <h1>Layout</h1>
      <Outlet />
    </div>
  }
>
  <Route path="/" element={<h2>Home</h2>} />
  <Route path="/about" element={<h2>About</h2>} />
</Route>
```
This can be simplified to cover just an example of a layout route. It doesn't need to be the full code.